.github/workflows/gh-pages.yml

```diff
@@ -23,7 +27,7 @@ jobs:
         run: asv publish
 
       - name: Deploy
-        uses: peaceiris/actions-gh-pages@v3
+        uses: peaceiris/actions-gh-pages@373f7f263a76c20808c831209c920827a82a2847 # v3
```

Comment:
This needs write permissions https://github.com/matplotlib/mpl-third-party/pull/192#issuecomment-3181488829